src/main/scala/KindProjector.scala

```diff
@@ -163,7 +163,7 @@ class KindRewriter(plugin: Plugin, val global: Global)
         val src = s"type _X_[$decoded] = Unit"
         sp.parse(src) match {
           case Some(TypeDef(_, _, List(tpe), _)) => tpe
-          case None => reporter.error(tree.pos, s"Can't parse param: $name"); null
+          case _ => reporter.error(tree.pos, s"Can't parse param: $name"); null
```

Comment:
Note to @SethTisue: since there's a proper code change here (it was necessary, otherwise the match is inexhaustive), I think this demands a `0.11.1` point release.

Reply:
Personally I don't see a need to be that strict about the version numbering, but okay. (You're in good company, I see some other maintainers, such as Olaf, being equally strict.)